packages/core/admin/admin/src/components/MainNav/NavUser.tsx

```diff
@@ -70,25 +94,41 @@ export const NavUser = ({ children, initials, ...props }: NavUserProps) => {
       <Menu.Root>
         <MenuTrigger endIcon={null} fullWidth justifyContent="center">
           <Avatar.Item delayMs={0} fallback={initials} />
-          <VisuallyHidden tag="span">{children}</VisuallyHidden>
+          <VisuallyHidden tag="span"> {getDisplayName(user) || 'Unknown User'}</VisuallyHidden>
```

Comment:
`getDisplayName` should fallback to the email. I don't think a user can exist without an email so I think it is safe to do:

```suggestion
          <VisuallyHidden tag="span"> {getDisplayName(user)}</VisuallyHidden>
```

Just a heads up for future PRs, if we did include the 'Unknown User' part we would need it to be translated 🙂

Reply:
understood!!